node/pkg/p2p/p2p.go

```diff
@@ -443,6 +443,11 @@ func processSignedHeartbeat(from peer.ID, s *gossipv1.SignedHeartbeat, gs *node_
 
 	digest := heartbeatDigest(s.Heartbeat)
 
+	// SECURITY: see whitepapers/0009_guardian_key.md
```

Comment:
I don't see this whitepaper in GH. I assume it will be coming soon?

Reply:
yes, PR: https://github.com/wormhole-foundation/wormhole/pull/1960